exporter/azure_monitor/example/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+	"log"
+	"go.opencensus.io/exporter/azure_monitor"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+func main() {
+

```

Comment:
curious - why do we have a blank line here?

Reply:
I forgot to delete that line after deleting other code. I will remove.